integration/base/src/main/java/com/inrupt/client/integration/base/AccessGrantScenarios.java

```diff
@@ -214,13 +213,13 @@ static void setup() throws IOException {
     @AfterAll
     static void teardown() {
          //cleanup pod
-        session = OpenIdSession.ofClientCredentials(
+        requesterSession = OpenIdSession.ofClientCredentials(
             URI.create(issuer), //Client credentials
-            CLIENT_ID,
-            CLIENT_SECRET,
+            REQUESTER_CLIENT_ID,
+            REQUESTER_CLIENT_SECRET,
```

Comment:
Shouldn't this be the resource owner session? If the access grant has been revoked, the requester will no longer have access.

Reply:
Good catch!